packages/ui/src/layout/banners/Announcement.Countdown.tsx

```diff
@@ -32,14 +47,10 @@ function CountdownBanner() {
     if (completed) {
       // Render a completed state
       return (
-        <div className="w-full flex gap-3 md:gap-6 items-center justify-center">
+        <div className='w-full flex gap-3 md:gap-6 items-center justify-center'>
```

Comment:
Can we please follow the same `prettier` rules as the rest of the project (ie. keep the double quotes for JSX)?